modules/preview-react/side-panel/lib/SidePanelToggleButton.tsx

```diff
@@ -46,31 +131,22 @@ export const SidePanelToggleButton = createComponent('button')({
 
     const rtlOrigin = useRTLOrigin();
```

Comment:
I think to avoid the type errors, you can update the type signature about this line to the following:

```tsx
const useRTLOrigin = (): 'left' | 'right' => {
      const isRTL = useIsRTL();
      // if the direction is set to RTl, flip the origin
      if (isRTL) {
        return context.origin === 'left' ? 'right' : 'left';
      }
      // Otherwise, default to returning the origin
      return context.origin;
    };
```

Reply:
You'll also have to update the `SidePanelContext` to the following:

```tsx
export const SidePanelContext = React.createContext({
  state: 'expanded',
  origin: 'left' as 'left' | 'right',
});
```